addons/binding/org.openhab.binding.wifiled/src/main/java/org/openhab/binding/wifiled/handler/LEDStateDTO.java

```diff
@@ -149,9 +154,9 @@ public LEDStateDTO withIncrementedProgramSpeed(int step) {
     }
 
     public Color getColor() {
-        float hue = (float) this.getHue().intValue() / 360;
-        float saturation = (float) this.getSaturation().doubleValue() / 100;
-        float brightness = (float) this.getBrightness().intValue() / 100;
+        float hue = (float) this.getHue().doubleValue() / 360.0f;
```

Comment:
In reality the problem is really : the usage of .intValue() that will do rounding before the calculation.  I was trying to be consistent with the rest of the code within this method and to ensure that calculations are done using the correct type. Since the end result needed to be float I tried to keep everything as float.
I am more than happy to change it to double...